JudoPayDotNetDotNet/JudoPaymentsFactory.cs

```diff
@@ -33,10 +33,8 @@ public static class JudoPaymentsFactory
 	    };
 
 
-        private static JudoPayApi Create(Credentials credentials, string baseUrl, IJudoConfiguration configuration)
+        private static JudoPayApi Create(Credentials credentials, string baseUrl, string apiVersion)
```

Comment:
that's interesting, I didn't know that InternalsVisibleTo allows you to also see private members